lib/vault.py

```diff
@@ -210,6 +210,28 @@ def configure(self):
                 client.write('/auth/aws-ec2/role/' + policy, policies = policy,
                                                              bound_iam_role_arn = arn_prefix + policy)
 
+        # AWS Authentication Backend
+        if aws_creds is None:
```

Comment:
Don't need this check anymore, as we want to remove usage of the `vault_aws_credentials` file